### PR TITLE
Update plugin.requirements hapi version to >=18.0.0

### PR DESCRIPTION
**Pull request by wswoodruff**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ exports.plugin = {
     multiple: true,
     pkg: require('../package.json'),
     requirements: {
-        hapi: '>=17.7.0'
+        hapi: '>=18.0.0'
```
Comment (devinivy) [inline lib/index.js]: Oh boy!  This actually doesn't need updating, does it?  This change would drop support of v17.

Comment (wswoodruff) [reply lib/index.js]: Ohp! Goodness, this was a mistake, definitely don't want to drop support for v17

**Comment by wswoodruff** [comment]
Closing. No need for this change, this change would drop support for v17 — not what we want. When v19 comes out in February we'll take a look and see if any code changes are necessary, and if there are breaking changes then we will update this

**Comment by wswoodruff** [comment]
We've been told there won't be any breaking changes in v19 so no need to think on it anymore

**Pull request closed without merging**